contentcuration/contentcuration/frontend/channelList/views/Channel/components/LanguageFilter.vue

```diff
@@ -68,7 +67,7 @@
     data() {
       return {
         languageInput: '',
-        availableLanguages: [],
+        availableLanguages: window.publicLanguages || [],
```

Comment:
If we have just passed the public language ids to the frontend we can do this before the component export:
```
import flatMap from 'lodash/flatMap';
import LanguagesMap, { LanguagesList } from 'shared/leUtils/Languages';

const publicLanguages = flatMap(window.publicLanguages, langId => {
  const baseLanguage = LanguagesMap[langId];
  return LanguagesList.filter(lang => lang.lang_code === baseLanguage.lang_code);
});
```
Can then replace this with:
`publicLanguages || [],`